packages/react-vapor/src/components/logoCard/LogoCard.tsx

```diff
@@ -89,11 +64,11 @@ export class LogoCard extends React.Component<ILogoCardProps & React.HTMLProps<H
                     <div className="logo-card-logo">
                         <Svg svgName={this.props.svgName} className={logoIconClassName} />
                     </div>
-                    <div className={logoCardContentClassName}>
+                    <div className="logo-card-content">
                         <h2 className="logo-card-title">{this.props.title}</h2>

```

Comment:
I think it looks better if we add the `mb1` class on the `h2` tag. What do you think?
![image](https://user-images.githubusercontent.com/35579930/105412338-96863580-5c02-11eb-8647-b6a7b86d4873.png)